### PR TITLE
Add an operation to Devices API

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description
- **What's changed?** Adds an operation to the Device API. See [Issue 3789](https://github.com/okta/okta-developer-docs/issues/3789).
- **Is this PR related to a Monolith release?** No.

### Netlify link

https://643f43f5fd2c2722a35aed44--reverent-murdock-829d24.netlify.app/docs/reference/api/devices/#list-all-users-for-a-device

### Resolves

* [OKTA-567622](https://oktainc.atlassian.net/browse/OKTA-567622)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/85868bad087fe8619335dee93c40fb9bce64dbfb/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/55a0161c-3d17-439b-b4f1-771c130667d5) | 86  | 96  | 83  | 57  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.9d7df1cd-9999-47fd-83fa-6b872460ef92?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 1]
```diff
@@ -180,20 +181,20 @@ Content-Type: application/json
...
+> **Note:** Listing devices with `search` shouldn’t be used as a part of any critical flows—such as authentication or updates—to prevent potential data loss. The `search` results may not reflect the latest information, as this endpoint uses a search index that may not be up to date with recent updates to the object. <br> Don't use search results directly for record updates, as the data might be stale and therefore overwrite newer data, resulting in data loss. <br> Use an `id` lookup for records that you update to ensure your results contain the latest data.
 
 This operation:
 
 * Supports pagination (to a maximum of 200 results).
 * Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding). For example, `search=profile.displayName eq "Bob"` is encoded as `search=profile.displayName%20eq%20%22Bob%22`.
 
-Searches include all Device profile properties, as well as the Device `id`, `status` and `lastUpdated` properties.
+Searches include all Device profile properties, and the Device `id`, `status` and `lastUpdated` properties.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:197]:
```suggestion
Searches include all Device profile properties, and the Device `id`, `status`, and `lastUpdated` properties.
```

**Review by brianduffield-okta — APPROVED** [review]
LGTM, one small nit

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/e431bc3e7dd54f6060c303a5b139414403009772/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f183d798-e93b-4dd0-8fd6-a590cd795ff1) | 86  | 96  | 83  | 57  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.05b56917-555e-4e5d-a860-84bc9d891df4?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**